substrate-node/pallets/pallet-kvstore/src/kvstore.rs

```diff
@@ -0,0 +1,24 @@
+use super::pallet::{Config, Error, Event, Pallet, TFKVStore};
+use frame_support::pallet_prelude::{ensure, DispatchResultWithPostInfo};
+use sp_std::prelude::Vec;
+
+impl<T: Config> Pallet<T> {
+    pub fn _set(user: T::AccountId, key: Vec<u8>, value: Vec<u8>) -> DispatchResultWithPostInfo {
```

Comment:
Do we have some tests for it?

Reply:
`set` function is almost wrapper for `_set` function. I see we have `tests::test_set_and_get` to test `set` function which should enough i guess.